.github/workflows/resize-images.yml

```diff
@@ -0,0 +1,27 @@
+name: Resize Images
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
I think that it should only happen on releases, though I don't know how to implement that.

Reply:
Well, what *is* a release? Do we always want to do one on merging to master?

Reply:
Automating the versioning in the correct way would be a pain. I don't think we'd want to do that.